pkg/operator/controller/ingress/controller_test.go

```diff
@@ -300,6 +349,29 @@ func TestSetDefaultPublishingStrategyHandlesUpdates(t *testing.T) {
 				LoadBalancer: lbs,
 			}
 		}
+		nlbScope = func(scope operatorv1.LoadBalancerScope, policy *operatorv1.LoadBalancerDNSManagementPolicy) *operatorv1.EndpointPublishingStrategy {
+			lbStrategy := lbs(scope, policy)
+			lbStrategy.ProviderParameters = &operatorv1.ProviderLoadBalancerParameters{
+				Type: operatorv1.AWSLoadBalancerProvider,
+				AWS: &operatorv1.AWSLoadBalancerParameters{
+					Type:                          operatorv1.AWSNetworkLoadBalancer,
+					NetworkLoadBalancerParameters: &operatorv1.AWSNetworkLoadBalancerParameters{},
+				},
+			}
+			return eps(lbStrategy)
+		}
+		classicScope = func(scope operatorv1.LoadBalancerScope, policy *operatorv1.LoadBalancerDNSManagementPolicy) *operatorv1.EndpointPublishingStrategy {
+			lbStrategy := lbs(scope, policy)
+			lbStrategy.ProviderParameters = &operatorv1.ProviderLoadBalancerParameters{
+				Type: operatorv1.AWSLoadBalancerProvider,
+				AWS: &operatorv1.AWSLoadBalancerParameters{
+					Type:                          operatorv1.AWSClassicLoadBalancer,
+					ClassicLoadBalancerParameters: &operatorv1.AWSClassicLoadBalancerParameters{},
+				},
+			}
+			return eps(lbStrategy)
+		}
```

Comment:
I wonder whether we can do without these functions.  The purpose of these helpers is to make the test cases easier to understand, but it's becoming confusing what the difference is between, say, `classicScope` and `elbWithNullParameters`.